src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/InputInProcess.cs

```diff
@@ -124,8 +125,13 @@ private async Task WaitNavigationItemShowsUpAsync(CancellationToken cancellation
             await TestServices.Workspace.WaitForAllAsyncOperationsAsync(new[] { FeatureAttribute.NavigateTo }, cancellationToken);
             // Since the all-in-one search experience populates its results asychronously we need
             // to give it time to update the UI. Note: This is not a perfect solution.
-            await Task.Delay(1000);
-            await WaitForApplicationIdleAsync(cancellationToken);
+            await TestServices.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
if we're in async code, do we need this RunAsync?  can we just inline the body of this here?

Reply:
Sure